rush.json

```diff
@@ -325,6 +325,11 @@
       "projectFolder": "sdk/purview/purview-scanning-rest",
       "versionPolicyName": "client"
     },
+    {
+      "packageName": "@azure-rest/purview-workflow",
+      "projectFolder": "sdk/purview/purview-workflow-rest",
+      "versionPolicyName": "client"
+    },
```

Comment:
have you run `rush update` locally? if yes, you will need to check in the pnpm-lock.yaml file

Reply:
Yes, I have run `rush update` but didn;t commit pnpm-lock.yaml file, have fixed it, thanks